### DISCUSSION TITLE
how to call component in console after click it in vue3 devtool, just like $vm0 in vue2 devtool?

### DISCUSSION THREAD

**Discussion by adoin** [vuejs/devtools-v6#1606] [Q&A] [answered]
### What problem does this feature solve?
vue3的devtool怎么像V2那时候一样，选定一个组件之后，控制台$vm0可以直接调用这个组件呢？ 我看到一个组件的id 不知道怎么用

### What does the proposed API look like?
e.g. console $vm0.listData I can get listData  after I inspect the component in devtool when Vue 2 version.
How to do this in Vue3

**Comment by Akryum** [comment, marked as answer]
Use `$vm.proxy.listData` or `$vm0.proxy.listData`
